docs/source/index.rst

```diff
@@ -12,6 +12,11 @@ Collects alerts for users in an inbox-like container
 
 Provides appointment scheduling and reminders
 
+:std:ref:`Atom Client <atom-client-module>`
+---------------------------------------------
+
+Fetches Atom feed(s) and publishes events when content changes
+
```

Comment:
I think this change actually needs to be made in the motech repo instead. There's a file at docs/source/modules/index.rst, which is actually the one rendered here: http://docs.motechproject.org/en/latest/modules/index.html